python_tests/test_python_frontend.py

```diff
@@ -1382,6 +1383,27 @@ def test_compute_contiguity(self):
         contiguity = [False, None, True, None, None, True, False]
         self.assertEqual(compute_contiguity(sizes, strides), contiguity)
 
+    def test_compute_tensor_descriptor(self):
+        sizes = [2, 1, 3, 1, 4, 3]
+        strides = [12, 4, 4, 4, 1, 0]
+        contiguity = [True, None, True, None, True, None]
+        stride_order = [5, 4, 3, 2, 1, 0]
+        computed_contiguity, computed_stride_order = compute_tensor_descriptor(
+            sizes, strides
+        )
+        self.assertEqual(computed_contiguity, contiguity)
+        self.assertEqual(computed_stride_order, stride_order)
+
+        sizes = [2, 3, 1, 5, 4]
+        strides = [28, 4, 14, 0, 1]
+        contiguity = [False, True, None, None, True]
```

Comment:
Shouldn't the innermost dim be the expanded dim, which has contiguity `None`?

Reply:
ha, this is where I wasn't quite sure about how we are representing the contiguity flag.

i.e. are contiguity flag corresponding to logical domain (rfactor domain), or should it be mapping to physical domain (allocation domain)?
Currently I'm viewing it as logical domain, does that make sense to you?

Reply:
I think it should be allocation domain?

Reply:
Thanks for confirming this one. I'll fix it~